src/modules/auth/store/authActions.store.ts

```diff
@@ -0,0 +1,48 @@
+import { PayloadAction, createAsyncThunk } from '@reduxjs/toolkit';
+import { signup, login } from '../api/index.api.ts';
+import { AuthState, TrainerSignup, Login } from '../types';
+
+export interface AuthStateOptions extends Partial<AuthState> {}
+
+export function updateAuthStateAction(
+  state: AuthState,
+  action: PayloadAction<AuthStateOptions>,
+) {
+  /**
```

Comment:
We can delete this comment, it was included in the `init` state, so it gives a little bit of context... 
Now it is just making a "mess"...